catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -392,6 +401,59 @@ public String getJson(Object result) {
     return objectMapper.toJson(result);
   }
 
+  public CqlQueryResponse executeCqlQuery(
+      CqlRequest cqlRequest,
+      CatalogFramework catalogFramework,
```

Comment:
`EndpointUtil` already has a `CatalogFramework`, and the `FilterAdapter` and `ActionRegistry` should be added to the `EndpointUtil` constructor instead.